ykrt/src/compile/jitc_yk/codegen/mod.rs

```diff
@@ -24,6 +24,8 @@ pub(crate) trait CodeGen: Send + Sync {
         m: Module,
         mt: Arc<MT>,
         hl: Arc<Mutex<HotLocation>>,
+        // Stack pointer offset from the base pointer of interpreter frame as defined in
```

Comment:
This needs to be `///`.

Reply:
Rust doesn't allow this: https://github.com/ykjit/yk/pull/1422#discussion_r1804388706

Reply:
Sorry, I misread the diff.

The standard way to do this is to put this in the function's docstring, not as a comment in the parameter list.

Reply:
[0d16998](https://github.com/ykjit/yk/pull/1422/commits/0d16998eeb2f3b539bf7169dd057680224781d55)